lib/mina-circle/circle-ci/client.rb

```diff
@@ -14,6 +14,7 @@ class CircleCI::Client
   def get(path, params = {})
     uri = URI("#{BASE_URI}/#{path}")
     uri.query = URI.encode_www_form(params) unless params.empty?
+    puts "Sending URL: #{uri}"
```

Comment:
Mina has an output helper which matches the format of other Mina output. https://github.com/mina-deploy/mina/blob/343a7ab672d8b4f0ddb84ec240cde7d94b46397a/lib/mina/output_helpers.rb#L22-L32